### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by ZooeyMiller** [FACG2/CodeBytes#13]
Your site is really nice for accessing via the keyboard (I can tab through really easily) but you're getting some warnings when I run an accessibility audit.

![codebytes - google chrome_006](https://user-images.githubusercontent.com/21139983/28153296-9aabd43e-679c-11e7-91d4-a1680e802b45.png)

If you're not already using them, the [accessibility developer tools](https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb?hl=en) are super helpful! They won't tell you _everything_ for accessibility but they are a good start, and making sure your site doesn't have any `severe` or `warnings` when you run the accessibility audit is always good.
